src/ProjectTemplates/test/template-baselines.json

```diff
@@ -13,6 +13,8 @@
         "Shared/MainLayout.razor",
         "Shared/NavMenu.razor",
         "Shared/SurveyPrompt.razor",
+        "wwwroot/appsettings.Development.json",
+        "wwwwroot/appsettings.json",
```

Comment:
I was hoping we could avoid having any `appsettings.*.json` files in the default templates. They would be empty, so we'd be better off not issuing an HTTP request for them. It's the presence of the files that causes them to be fetched at runtime.

So, could the changes in this file be reverted?

In order to tell the template engine not to emit the files at all in the non-auth cases, you'll need to update `src\ProjectTemplates\ComponentsWebAssembly.ProjectTemplates\content\ComponentsWebAssembly-CSharp\.template.config\template.json` and use an `exclude` option with the `NoAuth` condition (see the list of entries in the `modifiers` dictionary in that file).

Hope that makes sense, but if not, please ping me or someone else for clarification!

Reply:
Thanks for resurfacing this. I've addressed this now.